content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -6,7 +6,31 @@ further_reading:
   text: "Troubleshooting APM Connection Errors"
 ---
 
-## Enable debug mode
+## Automated debug log collection
+
+<div class="alert alert-warning">Automated debug logs are supported for Node.js and .NET only. For other languages, use <a href="/tracing/troubleshooting/tracer_debug_logs/#manual-debug-log-collection">manual debug log collection</a> instead.</div>
+
+A flare allows you to send necessary troubleshooting information to the Datadog support team, including tracer logs, with sensitive data removed. Flares are useful for troubleshooting issues like high CPU usage, high memory usage, and missing spans.
+
+The data collected varies by language. See [Data collected](#data-collected).
```

Comment:
Link doesn't work.

Reply:
@urseberry Oops, thank you, they had me remove that section recently. Updated to remove the reference!